packages/app/src/vis-packs/ValueLoader.tsx

```diff
@@ -32,7 +40,9 @@ function ValueLoader(props: Props) {
         <div />
         <div />
       </div>
-      <p>{message}...</p>
+      <p>
+        {message}... ({progress} %)
```

Comment:
I tried by adding the `Content-Length` headers in h5grove and it works !

I would simply suggest to round `progress` as now, it displays a number with many decimals.

Also, the `ValueLoader` displays `0%` just before disappearing which is not great. Perhaps due to the `removeProgressListener` ?

Reply:
Great! Okay, I've rounded the number and I've made sure the `progress` is reset when a new request starts rather than when a request fulfills.